src/server/oidc/controllers/authorize-controller.js

```diff
@@ -11,18 +10,11 @@ import { config } from '~/src/config/index.js'
 import { loginValidation } from '~/src/server/oidc/helpers/schemas/login-validation.js'
 import { registrationAction } from '~/src/server/registration/helpers/registration-paths.js'
 
-function getRequestUrl({ url, path }) {
-  const queryString = url.searchParams
-    ? qs.stringify(url.searchParams, { addQueryPrefix: true, encode: false })
-    : ''
-  return `${url.protocol}//${url.host}${path}${queryString}`
-}
-
 const authorizeController = {
   handler: async (request, h) => {
     const redirectUri = request.query?.redirect_uri
     if (config.get('oidc.showLogin') && request.query.user === undefined) {
-      const requestUrl = getRequestUrl(request)
+      const requestUrl = `${appBaseUrl}${request.path}${request.url.search}`
```

Comment:
`appBaseUrl` is being accesed in `oidc-config.js` and passed out. Feels like that can be removed from `oidc-config.js` and then here just get it from the app config?